src/components/CreateForm/CreateForm.js

```diff
@@ -48,6 +48,7 @@ const Form = styled.form`
   gap: 16px;
   background: rgba(250, 250, 250, 0.9);
   padding: 20px;
+  width: 335px;
 `
 const Heading = styled.h2`
   margin: 5px;
```

Comment:
maybe choose relative units instead of pixel?

Reply:
Since the form will not change with new input or other reasons, I choose to use absolute units. Thank you though!  👍🏻